src/types.ts

```diff
@@ -65,6 +65,7 @@ export type Dappeteer = {
   switchNetwork: (network: string) => Promise<void>;
   confirmTransaction: (options?: TransactionOptions) => Promise<void>;
   sign: () => Promise<void>;
+  signTypedData: () => Promise<void>;
```

Comment:
You should add in Dappeteer interface as well

Reply:
I'm not sure I understand. Do you mean the documentation? I couldn't find a Dappeteer interface other than this type.

Reply:
https://github.com/ChainSafe/dappeteer/blob/master/src/types.ts#L46
Not the best location or way to do this :/